proto/permission/exported/v1beta1/types.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package permission.exported.v1beta1;
+
+option go_package = "github.com/axelarnetwork/axelar-core/x/permission/exported";
+
+import "gogoproto/gogo.proto";
+
+option (gogoproto.goproto_getters_all) = false;
+
+enum Role {
+  option (gogoproto.goproto_enum_prefix) = false;
+  option (gogoproto.goproto_enum_stringer) = true;
+
+  ROLE_UNSPECIFIED = 0;
+  ROLE_GOVERNANCE = 1 [ (gogoproto.enumvalue_customname) = "Governance" ];
```

Comment:
can we find better names for the roles? It doesn't tell much to have a governance account with type governance and one with controller